src/codeu/chat/server/View.java

```diff
@@ -78,6 +78,81 @@ public View(Model model) {
   @Override
   public Message findMessage(Uuid id) { return model.messageById().first(id); }
 
+  @Override
+  public Collection<String> userStatusUpdate(String name, Uuid owner) {
```

Comment:
This function and `conversationStatusUpdate` are both nested quite a bit (few if blocks and for loops). Generally anything more than 2 nested blocks, it is always easier if you split them into smaller methods (you can keep them private if you do not want to expose them outside).